awx_collection/plugins/modules/export.py

```diff
@@ -86,11 +86,6 @@
         - workflow names, IDs, or named URLs to export
       type: list
       elements: str
-    applications:
```

Comment:
I did not remove the applications module from the Collection PR. I thought it was used for more purposes apart from Oauth. Is it not?

Reply:
I didn't think applications serve any purpose other than for creating tokens. IMO, the tests seemed to reflect this. It's not impossible that I'm wrong, but it seemed relatively clear even from reading the issue for _this_ PR.